resources/public/api/conf/1.2/application.yaml

```diff
@@ -361,14 +377,25 @@ paths:
           - tax-free-childcare-payments
       
 
-  /individuals/tax-free-childcare/payments/:
+  /individuals/tax-free-childcare/payments:
```

Comment:
please, add the header information to the endpoint.
Accept: application/vnd.hmrc.1.2+json